engine/src/test/java/org/hibernate/validator/test/constraintvalidator/ConstraintDefinitionContributorTest.java

```diff
@@ -48,85 +47,19 @@ public void constraint_definitions_can_be_configured_via_service_loader() {
 		assertCorrectConstraintTypes( constraintViolations, MustMatch.class );
 	}
 
-	@Test(expectedExceptions = UnexpectedTypeException.class, expectedExceptionsMessageRegExp = "HV000030.*")
```

Comment:
Because the concept of `ConstraintDefinitionContributor` does not exist anymore. Which was asked explicitely of me: "Let's not provide two different APIs to achieve the same thing, it's against the spirit of design we agreed long time ago.". So : 

 * `addConstraintDefinitionContributor` disappeared completely, so the tests around this method are not relevant anymore
 * `HibernateValidatorConfiguration.CONSTRAINT_DEFINITION_CONTRIBUTORS` disappeared too (`ConstraintMappingContributor`s may be used instead), so the tests around this property are not relevant anymore
 * I removed `validating_constraint_without_registered_constraint_validator_throws_exception` because it was already tested there: `org.hibernate.validator.test.internal.engine.valuehandling.UnwrapModesTest.no_constraint_validator_for_wrapped_and_unwrapped_value_throws_exception()`

I checked that `ValidatorConstraintMappingTest` covered the same features in the new API. I didn't see the point of having two places where I test the same thing.